### ISSUE TITLE
Does it works on ubuntu 20.04 with ROS noetic and python 3+?

### ISSUE THREAD

**Issue by Mdmorshadurrahman** [HumaRobotics/modbus#5]
Hello, 
I am working on one project regarding modbus and I tested it on ubuntu 16 but I have to successfully run on ubuntu 20.04 with ROS noetic and python 3+. Is there any developed version of it available to work according yo updated ubuntu and ROS versions?

**Comment by JadeCong** [comment]
@Mdmorshadurrahman Yeah, of course, but you may have to modify some files format from python version 2+ to version 3+.
Here is the modified solution from my project, then you can have a reference and try it.
Reference: https://github.com/JadeCong/siemens_ros/tree/main/modbus
